config/src/queue.rs

```diff
@@ -5,8 +5,8 @@ use serde::{Deserialize, Serialize};
 
 const DEFAULT_MAX_TRANSACTIONS_IN_QUEUE: u32 = 2_u32.pow(16);
 const DEFAULT_MAX_TRANSACTIONS_IN_SIGNATURE_BUFFER: u32 = 2_u32.pow(16);
-// 24 hours
-const DEFAULT_TRANSACTION_TIME_TO_LIVE_MS: u64 = 24 * 60 * 60 * 1000;
+/// Default length of time transaction will be kept if not committed
+pub const DEFAULT_TRANSACTION_TIME_TO_LIVE_MS: u64 = 24 * 60 * 60 * 1000; // 24 hours
```

Comment:
Might be a good idea to consider a stronger type that can be deserialized from JSON. E.g. 
```json
{
  "TRANSACTION_TIME_TO_LIVE_MS": {
    "hours": 24
  }
}
```

Reply:
maybe, @0x009922 can we include this suggestion in configuration discussions?

Reply:
For user convenience, we might use an approach from an incredibly popular [`ms` node package](https://github.com/vercel/ms):

```js
ms('2 days')  // 172800000
ms('1d')      // 86400000
ms('10h')     // 36000000
ms('2.5 hrs') // 9000000
ms('2h')      // 7200000
ms('1m')      // 60000
ms('5s')      // 5000
ms('1y')      // 31557600000
ms('100')     // 100
ms('-3 days') // -259200000
ms('-1h')     // -3600000
ms('-200')    // -200
```

Reply:
I'll close this then